src/RoboFile/ClientContainer.php

```diff
@@ -33,7 +33,11 @@
 	public function runContainerTests(
 		$opts = array(
 			'env' => 'desktop',
-			'debug' => false
+			'debug' => false,
+			'single' => false,
```

Comment:
the function already exists actually :) it's just a matter of how the main coordinator will execute the functions in the containers